.github/release-drafter.yml

```diff
@@ -0,0 +1,26 @@
+categories:
+  - title: 'Major Changes'
+    labels:
+      - 'major'  # c6476b
+  - title: 'Minor Changes'
+    labels:
+      - 'feature'  # 006b75
+      - 'enhancement'  # ededed
+      - 'performance'  # 555555
+  - title: 'Bugfixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+      - 'bug'  # fbca04
+      - 'docs'  # 4071a5
```

Comment:
How do you feel about moving `docs` into a new documentation section?

Reply:
IMHO too many sections makes it harder to read. IMHO 2-3 sections are more than enough.

Reply:
I agree in not wanting too many sections. But on the other hand it seems a bit misleading to label a documentation change a bug fix 

Thoughts @BeyondEvil ?